.circleci/config.yml

```diff
@@ -0,0 +1,114 @@
+version: 2.0
+
+install: &install
```

Comment:
what does the `&install` here mean? I'm not familiar with this syntax

Reply:
It is like a reference or label, so defined once and used many times.